apps_wishlist.md

```diff
@@ -1,168 +1,310 @@
+| [Geneweb](https://geneweb.tuxfamily.org) |  | [Upstream](https://github.com/geneweb/geneweb) |  |
+| GetSimple CMS |  | [Upstream](https://github.com/GetSimpleCMS/GetSimpleCMS) |  |
+| gitolite | Git server |  | [Package Draft](https://github.com/matlink/gitolite_ynh) |
+| gitrepositories |  |  | [Package Draft](https://github.com/YunoHost-Apps/gitrepositories_ynh) |
+| gitweb | Git forge (c.f. Gitolite) |  | [Package Draft](https://github.com/matlink/gitweb_ynh) |
+| [glitch-soc](https://glitch-soc.github.io/docs/) | A libre and federated social network |  | [Package Draft](https://github.com/Tagadda/glitch-soc_ynh) |
+| glpi | Create a beautiful blog or website easily |  | [Package Draft](https://github.com/abeudin/glpi_ynh) |
+| gnusocial | Create a federated comunication node |  | [Package Draft](https://github.com/YunoHost-Apps/gnusocial_ynh) |
+| [Goaccess](https://goaccess.io) | Web log analyzer | [Upstream](https://github.com/allinurl/goaccess) |  |
+| gogs_webhost | Turn a gogs repository to static web hosting |  | [Package Draft](https://github.com/YunoHost-Apps/gogs_webhost_ynh) |
+| Gollum |  | [Upstream](https://github.com/gollum/gollum) |  |
+| Goploader |  | [Upstream](https://github.com/Depado/goploader) |  |
+| Gpodder.net |  |  |  |
+| Granary |  | [Upstream](https://github.com/snarfed/granary) |  |
+| [Graphhopper](https://www.graphhopper.com/) |  | [Upstream](https://github.com/graphhopper/graphhopper) |  |
+| [Grocy](https://grocy.info/) | Fridge ERP | [Upstream](https://github.com/grocy/grocy) |  |
+| [Guacamole](https://guacamole.apache.org/) |  |  |  |
+| [Habitica](https://habitica.com/) |  | [Upstream](https://github.com/HabitRPG/habitica) |  |
+| headphones | Automatic music downloader |  | [Package Draft](https://github.com/YunoHost-Apps/headphones_ynh) |
+| Helpy |  | [Upstream](https://github.com/helpyio/helpy) |  |
+| [Hexo](https://hexo.io/) |  | [Upstream](https://github.com/hexojs/hexo) |  |
+| Hometown |  | [Upstream](https://github.com/hometown-fork/hometown) |  |
+| [htpc-manager](https://htpc.io) | Manage your HTPC from anywhere |  | [Package Draft](https://github.com/scith/htpc-manager_ynh) |
+| huginn | Build agents that monitor and act on your behalf |  | [Package Draft](https://github.com/YunoHost-Apps/huginn_ynh) |
+| [humhub](https://www.humhub.org) | Enterprise Social Network |  | [Package Draft](https://github.com/yunohost-apps/humhub_ynh) |
+| [Icecast 2](https://www.icecast.org) |  | [Upstream](https://gitlab.xiph.org/xiph/icecast-server/) |  |
+| [ikiwiki](https://ikiwiki.info) |  |  |  |
+| internetarchive | Offline version of the Internet Archive | [Upstream](https://github.com/internetarchive/dweb-mirror) | [Package Draft](https://github.com/mitra42/internetarchive_ynh) |
+| Invidious |  | [Upstream](https://github.com/omarroth/invidious) |  |
+| [Invoice Ninja](https://www.invoiceninja.com) |  | [Upstream](https://github.com/invoiceninja/invoiceninja) |  |
```

Comment:
```suggestion

```
We can remove InvoiceNinja out of the list as it is up and running https://github.com/YunoHost-Apps/invoiceninja_ynh